src/main/webapp/displayBankStatement.jsp

```diff
@@ -10,7 +10,7 @@
 <body style=background-color:orange;>
 <form action="displayStatementServlet" >
 <p>
-<h2 input style="margin-left: 30em";> Display Statement </h2> <br> <br>
+<h2 input style="margin-left: 30em";> Display Statement </h2> <br> <br> 
 <c:if test="${ failedDBConnection eq true}">
```

Comment:
- do not display unable to connect to database. make the message less verbose
- placeholder should have meaningful text, not exact value
- dont use multiple nbsp for putting spaces, manage it by css instead

Reply:
- do not display unable to connect to database. make the message less verbose
        **_"Bank Unreachable , check Internet Connection/try later"_**
- placeholder should have meaningful text, not exact value
        "Eta Edureka'r jonno korechi"
- dont use multiple nbsp for putting spaces, manage it by css instead